react/src/reactTextMask.js

```diff
@@ -66,7 +51,24 @@ export const MaskedInput = React.createClass({
       this.props.onChange(event)
     }
   }
-})
+}
+
+MaskedInput.propTypes = {
```

Comment:
proptypes are static methods and have to be added to classes like this unless if you want to add static prop types which isn't a standard in js yet